packages/admin-web/src/App.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import WelcomePage from './components/WelcomePage';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+// import Breadcrumb from './components/Breadcrumb';
```

Comment:
Left over from testing?

Reply:
no TBH, I kept it to remind us of having a breadcrumb.

Reply:
I'd personally lean towards adding the breadcrumb when needed, as otherwise our code can become polluted with commented-out pieces with implicit reasons.

If there's follow-up work (such as setting up breadcrumbs), let's track it with a ticket :+1:

Reply:
I added and updated the screenshot.